modules/vba_documents/app/models/vba_documents/upload_submission.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'vba_documents/sql_support'
+require 'vba_documents/upload_error'
```

Comment:
Somehow, the evidence-submissions endpoint will fail to find the class `VBADocuments::UploadError` when uploading without this line - I am not sure why this is necessary now